andela/mrm_api
# #163294072 Add room tags model

#### What does this PR do?
- Creates the room tags model

#### Description of Task to be completed?
The task is to create a room tags model. This is to allow for a uniform office structure across all locations.

#### How should this be manually tested?
- Clone the repo.
- Run `make build`
- `make run-app`
- Then run migrations
- You will see tags and room_tags tables added to the database.

**Any background context you want to provide?**
You may need to connect Postico to your database to view the created tables.

**What are the relevant pivotal tracker stories?**
[#163294072](https://www.pivotaltracker.com/story/show/163294072)

#### Screenshots


**Questions:**
None
